wp-content/themes/core/components/slider/css/slider.pcss

```diff
@@ -44,3 +74,87 @@
 		background: none;
 	}
 }
+
+/* -----------------------------------------------------------------------------
+ *
+ * Slider Arrows
+ *
+ * ----------------------------------------------------------------------------- */
+
+.c-slider__button--prev,
+.c-slider__button--next {
+
+	/* CASE: Dialog */
+	.c-dialog & {
+		display: none;
+		align-items: center;
+		justify-content: center;
+		top: 50%;
+		height: 50px;
+		width: 50px;
+		margin-top: 0;
+		border: 1px solid var(--color-white);
+		background: transparent;
+		color: var(--color-white);
+		transition: var(--transition);
+		transform: translateY(-50%);
+		opacity: 0.5;
+
+		&:before {
+			display: none;
+		}
+
+			&:after {
+			color: var(--color-white);
+			content: var(--icon-chevron-right);
+			font-family: var(--font-family-core-icons);
+			font-size: 22px;
+		}
+
+		&:focus,
+		&:hover {
+			background-color: var(--color-white);
+			opacity: 1;
+			transition: var(--transition);
+			
+
+			&:after {
+				color: var(--color-black);
+			}
+		}
+
+		&:focus {
+			outline: none;
+		}
+
+		@media (--viewport-full) {
+			display: flex;
+		}
+	}
+}
+
+.c-slider__button--next,
+.swiper-container-rtl .c-slider__button--prev {
+
+	/* CASE: Dialog */
+	.c-dialog & {
+		right: -102px;
```

Comment:
Will this value work okay on small screens? It doesn't create horizontal scroll bars by chance?